util/src/main/java/tech/pegasys/artemis/util/config/ArtemisConfiguration.java

```diff
@@ -399,4 +431,25 @@ public int getBeaconRestAPIPortNumber() {
   public boolean getBeaconRestAPIEnableSwagger() {
     return config.getBoolean("beaconrestapi.enableSwagger");
   }
+
+  private LoggingDestination getLoggingDestination() {
+    switch (config.getString("logging.destination")) {
+      case "consoleOnly":
+        return LoggingDestination.CONSOLE_ONLY;
+      case "fileOnly":
+        return LoggingDestination.FILE_ONLY;
+      case "both":
+      default:
+        return LoggingDestination.BOTH;
+    }
+  }
+
+  public LoggingConfiguration getLoggingConfiguration() {
```

Comment:
Probably should invert this dependency too so have `LoggingConfiguration.fromConfig(ArtemisConfiguration)`.

Reply:
Yeah, I would have done but ...circular dependency issues.
`ArtemisConfiguration` resides in the `artemis` module that has a dependency on the `logging` module. Referencing `ArtemisConfiguration` in `logging` would now require `logging` be dependant on `artemis`.

Unless I were to move `ArtemisConfiguration` into that conveniently named `config` module 🤔

Reply:
Turns out (after checking) `ArtemisConfiguration` is in `util` and not `artemis`, the inverting idea works 👍

Reply:
`ArtemisConfiguration` is in `util` currently.  So this would have `logging` depend on `util` rather than the other way around.  `config` is not actually a module, it's a directory that holds the config file. :) But I definitely wouldn't object to splitting `ArtemisConfiguration` out of `util` on the basis that `util` has way too much stuff in it.  Not entirely sure it's worth it.

The other possibility is to do what `NetworkConfig` currently does and just leave someone else to do the mapping ie https://github.com/PegaSysEng/teku/blob/9cd23ae4a876ca08b416349164d1c07b4412022f/services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java#L247

I do think there is value in keep `ArtemisConfiguration` working with primitives because it's going to get a major rewrite soon so the simpler it is the simpler that will be.

Reply:
> it's going to get a major rewrite soon

Excellent 👍

Reply:
> The other possibility is to do what `NetworkConfig` 

As the conversion is done only once, I'll follow the `NetworkConfig`  pattern